src/lang/en.json

```diff
@@ -246,6 +246,21 @@
             "Keen": "Keen",
             "Spirit": "Spirit",
             "Vital": "Vital"
+        },
+        "Currencies": {
+            "Spheres": {
+                "Name": "Spheres",
+                "Denominations": {
+                    "Diamond": "Diamond",
+                    "TierTwoGems": "Garnet/Heliodor/Topaz",
+                    "TierThreeGems": "Ruby/Smokestone/Zircon",
+                    "TierFourGems": "Amethyst/Sapphire",
+                    "Emerald": "Emerald",
+                    "Chips": "Chips",
+                    "Marks": "Marks",
+                    "Broams": "Broams"
+                }
+            }
```

Comment:
We need to avoid having any world specifics in the core system. The actual currency specifications will need to be done in the world modules. 

Of course, for local testing its useful to have a currency set up, but let's keep it out of the PR.